README.md

```diff
@@ -13,7 +13,7 @@ All files under ./docs/ are hosted on the [Tizen Docs site](https://docs.tizen.o
 
 ## NOTE
 
-To contribute, see the [Contributing Guide](CONTRIBUTING.md) and the [issues list](https://github.com/Samsung/tizen-docs/issues).
+To contribute,Feel free to see the [Contributing Guide](CONTRIBUTING.md) and the [issues list](https://github.com/Samsung/tizen-docs/issues).
```

Comment:
@pixiedevpraveen, thank you for taking the initiative to contribute to our documentation set. Unfortunately, we will not be able to accept this commit as the proposed change is of no value addition and there seems to be a typo as well.